server/data_common/data_adaptor.py

```diff
@@ -388,3 +388,18 @@ def get_last_mod_time(self):
         except RuntimeError:
             lastmod = None
         return lastmod
+
+
+class AppFeature(object):
+    def __init__(self, path, available=False, method="POST", extra={}):
+        self.path = path
+        self.available = available
+        self.method = method
+        self.extra = extra
```

Comment:
Tbh, all 4 variable names mean absolutely nothing to me. Is there a way these names can be more descriptive?

Reply:
it sounds like this code isnt being used by the client at all. I'm going to remove all of the code related to it in a separate pr just to make the diff clear if anything breaks.